frontends/p4/fromv1.0/converters.cpp

```diff
@@ -321,6 +321,68 @@ const IR::Type_StructLike *TypeConverter::postorder(IR::Type_StructLike *str) {
 ///////////////////////////////////////////////////////////////
 
 namespace {
+class FixupExtern : public Modifier {
```

Comment:
I will assume that there aren't many changes to review here, since this code seems to be just moved between files.
If you want me to read this carefully, please let me know.